website/addons/base/views.py

```diff
@@ -571,6 +571,7 @@ def addon_deleted_file(auth, node, error_type='BLAME_PROVIDER', **kwargs):
         'file_path': file_path,
         'file_name_title': file_name_title,
         'file_name_ext': file_name_ext,
+        'version_id': version.identifier,
```

Comment:
Yeah, I'm trying to fix that now so it will pass the Travis test. Someone suggested I rewrite the test because I added it, does that sound like it's along the right tracks?